tests/unit/test_dataclasses.py

```diff
@@ -152,3 +153,58 @@ class DummyContainer(SerializableContainer):
     data = DummyContainer.from_serialized(serialized)
     assert data.foo == ['unserialized-from']
     assert serialized['bar'] == 'should-never-change'
+
+
+def test_normalize_options(root_logger):
+    @dataclasses.dataclass
+    class DummyContainer(DataContainer):
+        foo: int = field(option='--foo', default=1)
+        bar: List[str] = field(option='--bar', default_factory=list, multiple=True)
+        baz: str = 'undefined'
+        qux: List[str] = field(option='--qux', default_factory=list, multiple=True)
+        corge: bool = field(option='--corge / --no-corge', default=True, multiple=True)
```

Comment:
Hmm, depends, could be. `--foo --no-foo --foo` might look pointless, but maybe there's a use for it. It is definitely a copy & paste issue rather than intention.